### PR TITLE
Recreate examples for mlserver 1.2.4

### PR DESCRIPTION
**Pull request by cliveseldon**

### CODE REVIEW

File: samples/examples/image_classifier/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+VERSION := $(shell grep seldonio/mlserver ../../../scheduler/Makefile | cut -d':' -f2)
```
Comment (sakoush) [inline samples/examples/image_classifier/Makefile:1]: should we move this under `samples/scripts/` for consistency?

File: scheduler/Makefile [hunk 2]
```diff
@@ -20,7 +20,7 @@ SCHEDULER_IMG ?= ${DOCKERHUB_USERNAME}/seldon-scheduler:${CUSTOM_IMAGE_TAG}
 TRITON_IMG ?= nvcr.io/nvidia/tritonserver:22.11-py3
 GPU_ENABLED ?= 0
 
-KIND_NAME=ansible
+KIND_NAME=seldon
```
Comment (sakoush) [inline scheduler/Makefile:23]: is that really intended?

**Review by sakoush — APPROVED** [review]
LGTM. Left some comments.

File: scheduler/Makefile (hunk 2 above)
Comment (cliveseldon) [reply scheduler/Makefile:23]: yes Rafal changed the defaults for ansible

File: samples/examples/image_classifier/Makefile (hunk 1 above)
Comment (cliveseldon) [reply samples/examples/image_classifier/Makefile:1]: maybe in a future PR - as need to discuss how these more complex examples are trained together

**Pull request merged**